sdstore/cache2disk.go

```diff
@@ -0,0 +1,166 @@
+package sdstore
+
+import (
+	"fmt"
+	"path/filepath"
+	"os"
+	"strings"
+	"io"
+	"io/ioutil"
+)
+
+/**
```

Comment:
documentation in go is just comment block which not necessarily have to follow jsdoc style, well, at least the types are already explicit

Reply:
yes

Reply:
done